arch/arm/src/common/Toolchain.defs

```diff
@@ -309,16 +309,10 @@ else
   # Wrong warning array subscript [0] is outside array bounds:
   # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=105523
 
-  ifeq ($(GCCVER),)
-    export GCCVER := $(shell $(CC) --version | grep gcc | sed -E "s/.* ([0-9]+\.[0-9]+).*/\1/" | cut -d'.' -f1)
-  endif
-
-  ifeq ($(GCCVER),12)
     ARCHOPTIMIZATION += --param=min-pagesize=0
```

Comment:
Seems now indentations should be updated as well. Also what about comment above?